src/control/lib/txtfmt/entity.go

```diff
@@ -0,0 +1,87 @@
+//
+// (C) Copyright 2019-2020 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+package txtfmt
+
+import (
+	"fmt"
+	"text/tabwriter"
+)
+
+// EntityFormatter is a variant of TableFormatter that can be used
+// for neatly displaying attributes of a single entity.
+type EntityFormatter struct {
+	TableFormatter
+}
+
+// Format generates an output string for the supplied table rows.
+// It includes a single subject header, and each row is printed as an
+// attribute/value pair.
+func (f *EntityFormatter) Format(table []TableRow) string {
+	if len(f.titles) > 0 && f.titles[0] != "" {
+		f.formatHeader()
+	}
+
+	for _, row := range table {
+		for key, val := range row {
+			fmt.Fprintf(f.writer, "%s\t%s\t\n", key, val)
```

Comment:
A minor suggestion: Might look better to have the key/value pairs formatted as "key:\tvalue\n" with a colon in between. Also don't need the trailing tab here - the TableFormatter did that because it didn't track number of columns printed to know when it was at the end of the line.

Reply:
I think it's actually good to include it though, because tabwriter uses it as a delimiter. If we switched to using right-justified output it wouldn't work correctly without the trailing tab:
https://golang.org/pkg/text/tabwriter/ (note the TrailingTab testcase)

I'll add the colon and see what I think... You're probably right that it would look better and might be a little more parseable with shell utils.

Reply:
I added it, but on the other side of the tab. That makes things line up nicely to my eye.
e.g.
```
title
-----
a       : b     
c       : d     
bananas : grapes
```

Reply:
Ah. Though I think switching to right-justified output is pretty unlikely in this case, isn't it?